Dockerfile

```diff
@@ -19,9 +19,23 @@ COPY Gemfile.lock Gemfile.lock
 
 COPY . .
 
+#######################################
+#Staging
```

Comment:
These comments probably aren't necessary since the layer name is essentially the label anyways. :)